Desktop_Interface/genericusbdriver.cpp

```diff
@@ -81,54 +81,53 @@ void genericUsbDriver::setFunctionGen(int channel, functionGenControl *fGenContr
        //// CH1 is AUX!!  CH2 is "MAIN"!!////
       //////////////////////////////////////
 
-    int length, maxLength, numDivides, maxDivides;
-    double freq, amplitude, offset;
-    unsigned char *samples;
+    int maxLength; // NOTE: unused
```

Comment:
Good catch (again, I'm not sure 2016 me thought it was a good idea to avoid compiler warnings!).
This can be deleted, though.